modules/custom/az_news/az_news_export/src/Plugin/views/row/AZNewsDataFieldRow.php

```diff
@@ -35,17 +44,35 @@ class AZNewsDataFieldRow extends DataFieldRow {
    */
   protected $token;
 
+  /**
+   * A static cache of the types of entities that can be serialized.
+   *
```

Comment:
This list of target types for entity reference field types.  I'm not sure if the correct value is paragraph or paragraph_item at the moment.

Reply:
I think you probably just need `paragraph`.  That's the actual entity type ID.  `paragraphs_item` is the name of the DB table it uses.